Synview-backend/routes/user_routes.ts

```diff
@@ -115,9 +116,9 @@ userRouter
       context.response.body = {
         token: access_token,
       };
-
-      context.state.session.set("Authorization", `Bearer ${access_token}`);
-      context.response.headers.set("Authorization", `Bearer ${access_token}`);
+      context.cookies.set("Authorization", `Bearer ${access_token}`, {
+        expires: new Date(Date.now() + 60 * 60 * 1000),
```

Comment:
You are adding this expiration timestamp into the token, but do you enforce it anywhere?